action.yml

```diff
@@ -24,5 +24,22 @@ inputs:
     default: main
 
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
+  using: "composite"
+  steps:
+    - name: Set up pixi
+      uses: prefix-dev/setup-pixi@ba3bb36eb2066252b2363392b7739741bb777659  # v0.8.1
```

Comment:
I need to look at what the [`prefix-dev/setup-pixi`](https://github.com/prefix-dev/setup-pixi) action is actually doing to see if pinning at the hash level actually provides any safeguards. I'll also ask the `pixi` team to review this PR once it is ready to move from draft to review.

Reply:
the setup-pixi action is designed to be backwards compatible (like pixi as well) but there are no guarantees yet.

with a proper dependabot configuration and automatic updates, i would just keep it pinned to the SHA level, though for additional reproducibility